lib/SystemStatistics.php

```diff
@@ -54,8 +54,20 @@ public function __construct(IConfig $config, IAppManager $appManager, Installer
 	public function getSystemStatistics(): array {
 		$processorUsage = $this->getProcessorUsage();
 		$memoryUsage = $this->os->getMemory();
+		$updateInfo = [];
+		$updateInfo['lastupdatedat'] = (int) $this->config->getAppValue('core', 'lastupdatedat');
```

Comment:
`lastupdatedat` is useful to verify that the update check was run recently, and the information isn't totally outdated. I'm open to better ideas how to name it, since it might be unclear what was last updated, the server or the information about updates?  (see comment below for indepth response)